Sources/Basics/Observability.swift

```diff
@@ -161,6 +162,14 @@ public protocol DiagnosticsHandler: Sendable {
     func handleDiagnostic(scope: ObservabilityScope, diagnostic: Diagnostic)
 }
 
+//helper protocol to configurate the style of diagnostics
+protocol SeverityConfig {
+    var prefix: String { get }
+    var color: TerminalController.Color { get }
+    var isBold: Bool { get }
+}
+
+
 // helper protocol to share default behavior
```

Comment:
Let's convert this to a doc comment while we're at it.

```suggestion
/// Helper protocol to share default behavior.
```